script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -146,7 +147,7 @@ I'm not entirely sure why I lost my composure this badly.
 [sha:b72a7e8c027812a57da92302aa8f6cb70f78e5c7]{
 -- Page 42, Offset 14910.
 -- 先輩がいる事の驚きと、先輩がいる事の喜び。
-I'm both surprised to run into Senpai here, and overjoyed to see her.
+I'm both surprised to run into Senpai here and overjoyed to see her.
```

Comment:
revert

Reply:
This one definitely doesn't need a comma. Suppose it could be "I'm both surprised and overjoyed to run into Senpai here."